doc/vxlan/EVPN/EVPN_VXLAN_HLD.md

```diff
@@ -266,14 +262,6 @@ __Figure 3: Symmetric IRB packet flow__
 Both asymmetric and symmetric IRB models will be supported in SONiC.
 
 
-
-#### ARP and ND Suppression
```

Comment:
From the changes, my understanding is that you tried to remove previous ARP / ND suppression changes? Is that correct?

Reply:
@eddieruan-alibaba yes that's correct